spec/counter_spec.rb

```diff
@@ -1,53 +0,0 @@
-require 'spec_helper'
-
-describe Prorate::Counter do
```

Comment:
The idea behind having a counter was also to be able to give decent info at some stage regarding the duration of the throttle and the requests used versus quota (see how the GH API operates). Granted - I didn't use that in the monolith, but it is one of the things which are really nice to have. Does the Lua setup support such a use case? (see how the GH API reports limits and throttling via headers)